contracts/governence/FedToken.sol

```diff
@@ -0,0 +1,18 @@
+//SPDX-License-Identifier: UNLICENSED
+
+// Solidity files have to start with this pragma.
+// It will be used by the Solidity compiler to validate its version.
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/presets/ERC20PresetMinterPauser.sol";
+
+
+contract FedToken is ERC20PresetMinterPauser {
```

Comment:
Could you add some comments about what this contract is and what it's supposed to do?

Reply:
will had in the future